server/controllers/group.js

```diff
@@ -5,6 +5,7 @@ import groupService from '../services/group';
 function getGroup(req, res) {
   req.group.populate('users')
     .execPopulate()
+    .then(updatedGroup => updatedGroup.populate('joinRequests').execPopulate())
```

Comment:
You can join this into the other populate line http://mongoosejs.com/docs/populate.html

`req.group.populate('users joinRequests')`